testing/demo/pkg/setup/initialise_groth16_client.go

```diff
@@ -59,6 +59,9 @@ func InitializeGroth16LightClientOnSimapp() error {
 }
 
 func createClientAndConsensusState(genesisBlock, latestBlock *ethtypes.Block) (*cdctypes.Any, *cdctypes.Any, error) {
+	// TODO: this clientState isn't created with a state transition verifier key or state inclusion verifier key.
```

Comment:
what do we need this for again?

Reply:
We need to provide a state transition verifier key so that the EVM light client deployed on SimApp can be updated.